cfn-resources/datalakes/mongodb-atlas-datalakes.json

```diff
@@ -544,38 +544,34 @@
       "type": "string",
       "description": "Human-readable label that identifies the Federated Database to remove."
     },
-    "Hostnames":{
-      "type":"array",
+    "Hostnames": {
+      "type": "array",
       "insertionOrder": false,
       "description": "Human-readable label that identifies the Federated Database to update.",
-      "items":{
-        "type":"string"
+      "items": {
+        "type": "string"
       }
     },
-    "State":{
-      "type":"string",
+    "State": {
+      "type": "string",
       "description": "Human-readable label that identifies the Federated Database to update."
     }
-
   },
-  "readOnlyProperties": [
-    "/properties/GroupId",
-    "/properties/DataProcessRegion",
+  "primaryIdentifier": [
+    "/properties/ProjectId",
     "/properties/TenantName",
+    "/properties/Profile"
+  ],
+  "createOnlyProperties": [
+    "/properties/ProjectId",
+    "/properties/TenantName"
+  ],
+  "readOnlyProperties": [
     "/properties/Storage",
-    "/properties/SkipRoleValidation",
     "/properties/StartDate",
     "/properties/Hostnames",
     "/properties/State",
-    "/properties/EndDate",
-    "/properties/CloudProviderConfig/Aws/TestS3Bucket",
-    "/properties/NDSDataLakeDatabaseCollectionView",
-    "/properties/DataLakeDatabaseDataSourceView",
-    "/properties/DataLakeDatabaseView",
-    "/properties/NDSDataLakeDatabaseView",
-    "/properties/DataLakeAWSCloudProviderConfigView",
-    "/properties/DataLakeStorageView",
```

Comment:
why did you remove these props from the read only?

Reply:
@andreaangiolillo  We don't have all these values from server, except Aws/TestS3Bucket and this property is sent from Client.

Hence removed which are not coming from Server.